### ISSUE TITLE
Error When Opening XLSX Files

### ISSUE THREAD

**Issue by AUH2O** [shuchkin/simplexlsxgen#9]
I have the following code snippet:
\
`$books = [
    ['ISBN', 'title', 'author', 'publisher', 'ctry' ],
    [618260307, 'The Hobbit', 'J. R. R. Tolkien', 'Houghton Mifflin', 'USA'],
    [908606664, 'Slinky Malinki', 'Lynley Dodd', 'Mallinson Rendel', 'NZ']
];

$xlsx = SimpleXLSXGen::fromArray( $books );
$xlsx->saveAs('books.xlsx');
$xlsx->downloadAs('books.xlsx');
`

The file is downloaded and save to my hard drive just fine. When I go to open it in Excel 2016 I get the following error message from Excel: "We found a problem with some content in books.xlsx. Do you want us to try and recover as much as we can." I can click on Yes and all the content is there and OK. What do I need to do to download Excel files and not get this message? If I use FileZilla and copy it to my hard drive from my website, I do not get the error message.

**Comment by AUH2O** [comment]
I found the problem, all the examples showed a carriage return between the "[ [" and "] ]" brackets so I figured they were needed. After posting this question I decided to do a test without a carriage return and there was no error message opening up the Excel file. 

Kudos to the author - a great simple little utility that I was looking for. Now to expand it to read a database file and save the rows from it.
